### PR TITLE
am62x-sk: Add addidtional flags for optee-os

### PR DESCRIPTION
**Pull request by paresh-bhagat12**
Enable additional flags for optee-os
- Enable SOFTWARE_PRNG to enable software based random number genrator. This also fixes low power mode failure on buildroot image.
- Set TEE_CORE_LOG_LEVEL to 1 for debug purpose.

### CODE REVIEW

File: configs/ti_release_am62x_sk_defconfig [hunk 1]
```diff
@@ -39,6 +39,7 @@ BR2_TARGET_OPTEE_OS_CUSTOM_REPO_URL="https://github.com/OP-TEE/optee_os.git"
 BR2_TARGET_OPTEE_OS_CUSTOM_REPO_VERSION="8f645256efc0dc66bd5c118778b0b50c44469ae1"
 BR2_TARGET_OPTEE_OS_NEEDS_PYTHON_CRYPTOGRAPHY=y
 BR2_TARGET_OPTEE_OS_PLATFORM="k3-am62x"
+BR2_TARGET_OPTEE_OS_ADDITIONAL_VARIABLES="CFG_WITH_SOFTWARE_PRNG=y CFG_TEE_CORE_LOG_LEVEL=1"
```
Comment (jsuhaas22) [inline configs/ti_release_am62x_sk_defconfig:42]: In Debian and Arago, we have only `CFG_TEE_CORE_LOG_LEVEL` for AM62x. `CFG_WITH_SOFTWARE_PRNG` is an argument only for AM62px.

**Review by jsuhaas22 — CHANGES_REQUESTED** [review]

**Review by cshilwant — CHANGES_REQUESTED** [review]
s/genrator/generator in commit message

**Review by cshilwant — CHANGES_REQUESTED** [review]
@paresh-bhagat12 

Better to explicitly mention linux as RT + LPM story is still not cracked

"This also fixes low power mode failure on buildroot `linux` image."

**Review by cshilwant — CHANGES_REQUESTED** [review]
s/debuggin/debugging

**Review by cshilwant — APPROVED** [review]

**Review by jsuhaas22 — APPROVED** [review]
LGTM

**Pull request merged**